src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -91,7 +91,7 @@ const UsersFragment = ({
   const [client] = React.useContext(Context).client;
   const {exhangeFollower, users, updateFollowDiscoveryContext} = useDiscovery();
   const [loadingDM, setLoadingDM] = React.useState(false);
-  const {createSignChat} = useCreateChat();
+  const {createSignChat, loadingCreateChat} = useCreateChat();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `loadingCreateChat` state variable from `useCreateChat()` hook is a good approach to handle the loading state of chat creation. However, it's important to ensure that this state is properly managed within the `useCreateChat()` hook to avoid any inconsistencies.
```diff
+ const {createSignChat, loadingCreateChat} = useCreateChat();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->